Neo4j.Driver/Neo4j.Driver.IntegrationTests/DirectDriver/BookmarkIT.cs

```diff
@@ -96,7 +97,7 @@ public void ShouldThrowForInvalidBookmark()
         {
             if (Skip) return;
             var invalidBookmark = "invalid bookmark format";
-            using (var session = Driver.Session())
+            using (var session = (Session)Driver.Session())
```

Comment:
To avoid compiling warnings :) The methods are not deprecated on Session class, they are deprecated on ISession